ports/unix/Makefile

```diff
@@ -91,6 +91,17 @@ ifndef DEBUG
 CFLAGS += -U _FORTIFY_SOURCE
 endif
 
+# Optional Undefined Behavior Sanitizer
```

Comment:
In a display of pedantry befitting a computer programmer, I've used US English spelling for the proper noun Undefined Behavior Sanitizer but used British English spelling elsewhere. If this is too silly then I can change it.